### ISSUE TITLE
Wechat Login Support with Authing.cn

### ISSUE THREAD

**Issue by zixia** [zixia/auth-angular#14]
That would be wonderful if we could support wechat login for `auth-angular`!

@leinue

**Comment by leinue** [comment]
I will be supporting this week.

**Comment by zixia** [comment]
Glad to hear that, looking forward to the supporting from Authing.cn!

**Comment by leinue** [comment]
I haven't used angular for several years. How can I debug this program, is there any documentation?

**Comment by zixia** [comment]
I'm afraid that you have to read the source code as documentation.

And please have a look at https://github.com/chatie/app , it consumes the auth-angular module and you can use that as a example.

**Comment by leinue** [comment]
ok

**Comment by leinue** [comment]
I underestimated the complexity of integrating into auth-angular and estimated that there will be a little more time.

**Comment by zixia** [comment]
Yes, to use either authing.cn or auth0.com is a too complicated work for developers, even he is familiar with authing.cn or auth0.com. That's why I want to write this module ...
